setup.cfg

```diff
@@ -23,84 +23,84 @@ classifiers =
 [options]
 include_package_data = True
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.8
 zip_safe = False
 install_requires =
     # Invenio core modules
-    invenio-app>=1.3.2,<1.4.0
-    invenio-base>=1.2.10,<1.3.0
-    invenio-cache>=1.1.0,<1.2.0
-    invenio-celery>=1.2.3,<1.3.0
+    invenio-app>=1.3.4,<1.4.0
+    # upper pinned to minor due to flask 2.3 upgrade changes dependencies
+    invenio-base>=1.2.11,<1.3.0
+    invenio-cache>=1.1.1,<2.0.0
+    invenio-celery>=1.2.4,<1.3.0
     invenio-config>=1.0.3,<1.1.0
-    invenio-i18n>=1.3.1,<1.4.0
+    invenio-i18n>=2.0.0,<3.0.0
     # Invenio base bundle
-    invenio-admin>=1.3.1,<1.4.0
-    invenio-assets>=1.2.7,<1.3.0
-    invenio-formatter>=1.1.3,<1.2.0
-    invenio-logging>=1.3.0,<1.4.0
-    invenio-mail>=1.0.2,<1.1.0
-    invenio-rest>=1.2.7,<1.3.0
-    invenio-theme>=1.3.10,<1.4.0
+    invenio-admin>=1.4.0,<1.5.0
+    invenio-assets>=3.0.0,<4.0.0
+    invenio-formatter>=2.0.0,<3.0.0
+    invenio-logging[sentry_sdk]>=2.0.0,<3.0.0
+    invenio-mail>=2.0.0,<3.0.0
+    invenio-rest>=1.3.0,<1.4.0
+    invenio-theme>=2.0.0,<3.0.0
     # Invenio auth bundle
-    invenio-access>=1.4.2,<1.5.0
-    invenio-accounts>=1.4.8,<1.5.0
-    invenio-oauth2server>=1.3.4,<1.4.0
-    invenio-oauthclient>=1.4.4,<1.5.0
+    invenio-access>=2.0.0,<3.0.0
+    invenio-accounts>=3.0.0,<4.0.0
+    invenio-oauth2server>=2.0.0,<3.0.0
+    invenio-oauthclient>=3.0.0,<4.0.0
     invenio-userprofiles>=1.2.4,<1.3.0
     # Invenio metadata bundle, without records UI
-    invenio-indexer>=1.3.0,<1.4.0
-    invenio-jsonschemas>=1.1.3,<1.2.0
-    invenio-oaiserver>=1.3.0,<1.4.0
-    invenio-pidstore>=1.2.2,<1.3.0
-    invenio-records-rest>=1.9.0,<1.10.0
-    invenio-records>=1.6.0,<1.7.0
+    invenio-indexer>=2.2.0,<3.0.0
+    invenio-jsonschemas>=1.1.4,<1.2.0
+    invenio-oaiserver>=2.2.0,<2.3.0
+    invenio-pidstore>=1.3.0,<1.4.0
+    invenio-records-rest>=2.2.0,<3.0.0
+    invenio-records>=2.2.1,<3.0.0
     # Invenio files bundle, with only invenio-files-rest
-    invenio-files-rest>=1.2.0,<1.3.0
+    invenio-files-rest>=2.0.0,<3.0.0
     # Extra deps of ILS
     invenio-banners>=1.0.0a1,<1.1.0
-    invenio-circulation>=2.0.0a1,<3.0.0
-    invenio-opendefinition>=1.0.0a10,<1.1.0
     invenio-pages>=1.0.0a5,<1.1.0
-    invenio-pidrelations>=1.0.0a7,<1.1.0
-    invenio-stats>=2.0.0,<2.1.0
+    invenio-circulation>=3.0.0a1,<4.0.0
+    invenio-opendefinition>=2.0.0a1,<3.0.0
+    invenio-pidrelations>=0.2.0,<1.1.0
+    invenio-stats>=4.0.0,<5.0.0
     Flask-Debugtoolbar>=0.10.1,<1.0.0
     pycountry>=20.0.0,<23.0.0
     # needed to have namedtuple json serialized as dict
     simplejson>=3.8.1,<4.0.0
     pluggy>=0.13.1,<1.0.0
     # breaking changes in version 4
-    jsonschema>=3.0.0,<4.0.0
-    # version conflict by invenio-oauth2server
-    WTForms>=2.3.3,<3.0.0
-    # invenio-celery conflict fix
-    celery>=5.1.0,<5.2
+    jsonschema>=4.3.0,<5.0.0
```

Comment:
can you remove other pinned packages? Or are the others still needed?

Reply:
other packages seem to be needed still, I removed as much as I could